storage/s3-sdk/s3_sdk_test.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2021 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+
+import backoff
+from botocore.exceptions import ClientError
+from google.cloud import storage
+import pytest
+
+import list_gcs_buckets
+import list_gcs_objects
+
+
+PROJECT_ID = os.environ["MAIN_GOOGLE_CLOUD_PROJECT"]
```

Comment:
Can `_, project_id = google.auth.default()` be used to detect the project?

Reply:
What project will that return? We need to use `MAIN_GOOGLE_CLOUD_PROJECT` rather than the standard one because we need a permissions exception to have HMAC keys.